source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -149,6 +151,8 @@ SplitRequestPtr SimpleRequest::create(Router& router,
     request_ptr->handle_ =
         makeSingleServerRequest(route, base_request->asArray()[0].asString(),
                                 base_request->asArray()[1].asString(), base_request, *request_ptr);
+  } else {
+    ENVOY_LOG(debug, "redis: route not found: '{}'", incoming_request->toString());
```

Comment:
nit: is "redis:" prefix needed since you are using a redis log prefix already? Same below?

Reply:
I was n't sure as it is being used in other debugs in same file. But I think we can remove - Removed at all places to be consistent.